win32ss/include/ntuser.h

```diff
@@ -726,6 +726,8 @@ typedef struct _WND
     PSBINFOEX pSBInfoex; // convert to PSBINFO
     /* Entry in the list of thread windows. */
     LIST_ENTRY ThreadListEntry;
+
+    PVOID DialogPointer;
```

Comment:
Shouldn't we use a specific struct for Dialogs (or just say that this is behind it) ? You're making every allocated WND bigger but only dialogs use it. Or would it be erased for e.g. buttons ?

Reply:
The DIALOGINFO structure is only defined in dialog.c. Windows also stores it in the WND structure.

Reply:
Fine